src/screens/EditTextScreen.tsx

```diff
@@ -17,8 +17,10 @@ export const EditTextScreen = (props: Props) => {
   const { params } = props.route;
```

Comment:
Wouldn't it better to destructure `this.route.params` here, just in one place? It's hard to read one it's in multiline
```
const { label, header, onSave, title, maxLength, validate, validateOnSave, emptyValueAllowed } = props.route.params;
```

in this case a default value can be assigned this way:
```
const { label, header, onSave, title, maxLength, validate, validateOnSave, emptyValueAllowed = false } = props.route.params;
```